jsapp/js/account/stripe.types.ts

```diff
@@ -72,7 +72,7 @@ interface SubscriptionSchedule {
 }
 
 export interface SubscriptionInfo {
-  plan: PlanInfo;
+  plan: number;
```

Comment:
FYI, I noticed while working on this that plan here returns as the PK of the plan, rather than a nested Plan field. I figured the interface was probably just outdated so changed it accordingly.

Reply:
Good call! [Plans are deprecated in favor of prices](https://stripe.com/docs/api/plans), so we avoid using them.